ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py

```diff
@@ -74,6 +74,47 @@ def _get_dtype_and_matrix(draw):
         )
     )
 
+#norm
```

Comment:
comment also count as code, so keep two empty lines before it. also, you need a space between # and norm